### DISCUSSION TITLE
How can I define a non-enzymatic hydrolysis?

### DISCUSSION THREAD

**Discussion by OliversLab** [Open-Systems-Pharmacology/Forum#1660] [How-Tos] [answered]
Hi everybody,
The drug I seek to model undergoes non-enzymatic hydrolysis and I need to include the metabolite (product from hydrolysis), which is cleared by renal filtration, in the model as well. 

Is there a way to include this process in the default model? If not, how can this be done using Moby?

Thank you very much!

best regards,
Oliver

**Comment by StephanSchaller** [comment, marked as answer]
Hi Oliver,

Certainly, however, you would still implement this with a dummy enzyme uniformly expressed across all compartments/tissues where hydrolysis might happen. You can then link the parent and metabolite model through this dummy enzyme and define a renal CL for the metabolite.

Best,
Stephan

**Reply by OliversLab** [reply]
Hi Stephan,
thank you very much.
Do you happen to have an example in mind were this has been used before?

kind regards,
Oliver

**Reply by StephanSchaller** [reply]
Not sure about a public one, but yes, we did this before for client projects.
We have an example published without inclusion of a metabolite: https://bpspubs.onlinelibrary.wiley.com/doi/full/10.1111/bcp.15081
